docs/source/nodes_and_pipelines/micro_packaging.md

```diff
@@ -0,0 +1,122 @@
+  * To place parameters from a different config environment, run `kedro pipeline pull <pipeline_name> --env <env_name>`
+  * Pipeline unit tests in `src/tests/pipelines/<pipeline_name>`
+* Kedro will also parse any requirements packaged with the modular pipeline and add them to project level `requirements.in`.
+* It is advised to do `kedro build-reqs` to compile the updated list of requirements after pulling a modular pipeline.
+
+```eval_rst
+.. note::  If a modular pipeline has embedded requirements and a project ``requirements.in`` file does not already exist, it will be generated based on the project ``requirements.txt`` before appending the modular pipeline requirements.
+```
+
+You can pull a modular pipeline from different locations, including local storage, PyPI and the cloud:
+
+```eval_rst
++--------------------------------+--------------------------------------------------------------------------------------+
+| Operation                      | Command                                                                              |
++================================+======================================================================================+
+| Pulling from a local directory | ``kedro pipeline pull <project-root>/src/dist/<pipeline_name>-0.1-py3-none-any.whl`` |
++--------------------------------+--------------------------------------------------------------------------------------+
+| Pull from cloud storage        | ``kedro pipeline pull s3://my_bucket/<pipeline_name>-0.1-py3-none-any.whl``          |
++--------------------------------+--------------------------------------------------------------------------------------+
+| Pull from PyPI like endpoint   | ``kedro pipeline pull <pypi-package-name>``                                          |
++--------------------------------+--------------------------------------------------------------------------------------+
+```
+
+### Providing `fsspec` arguments
+
+* If you are pulling the pipeline from a location that isn't PyPI, Kedro uses [`fsspec`](https://filesystem-spec.readthedocs.io/en/latest/) to locate and pull down your pipeline.
+* You can use the `--fs-args` option to point to a YAML that contains the required configuration.
+
+```bash
+kedro pipeline pull https://<url-to-pipeline.whl> --fs-args pipeline_pull_args.yml
+```
+
+```yaml
+# `pipeline_pull_args.yml`
+client_kwargs:
+  headers:
+    Authorization: token <token>
+```
+
+## Pull multiple modular pipelines
+
+* To pull multiple modular pipelines in bulk, run `kedro pipeline pull --all`.
+* This will pull and unpack all pipelines specified in the `tool.kedro.pipeline.pull` manifest section of the project's `pyproject.toml` file:
+
+```toml
+[tool.kedro.pipeline.pull]
+"src/dist/first-pipeline-0.1-py3-none-any.whl" = {}
+"https://www.url.to/second-pipeline.whl" = {alias = "aliased_pipeline", fs-args = "pipeline_pull_args.yml"}
```

Comment:
No whl. Please see release notes on develop, there's some examples in the migration guide for 0.18.